### PR TITLE
finish tabletLandscape

### PR DESCRIPTION
**Pull request by nmelnychenko**

### CODE REVIEW

File: src/js/templates/ThirdBlock.js [hunk 1]
```diff
@@ -35,3 +36,23 @@ thirdBlockCircleSvg.forEach(element => {
...
+
+const scrollEvents = ['wheel', 'scroll', 'keydown']
+
+const addScroll = () => {
+    document.body.style.overflow = 'auto';
+}
+
+const preventScroll = () => {
+    document.body.style.overflow = 'hidden';
+    console.log('event', event)
```
Comment (StadniukYevhen) [inline src/js/templates/ThirdBlock.js]: remove `console.log`

File: src/js/templates/ThirdBlock.js [hunk 2]
```diff
@@ -35,3 +36,23 @@ thirdBlockCircleSvg.forEach(element => {
...
+    thirdBlockScrolledText.addEventListener('mouseleave', () => {
+        addScroll()
+    })
+}
+
+scrollEvents.forEach(item => thirdBlockScrolledText.addEventListener(item, (event) => {
```
Comment (StadniukYevhen) [inline src/js/templates/ThirdBlock.js]: move `addEventListener` to the next line

Comment (nmelnychenko) [reply src/js/templates/ThirdBlock.js]: done

File: src/js/templates/ThirdBlock.js (hunk 1 above)
Comment (nmelnychenko) [reply src/js/templates/ThirdBlock.js]: done

**Pull request merged**